### ISSUE TITLE
Add SIL logo to installers and splash screens

### ISSUE THREAD

**Issue by mcdurdin** [keymanapp/keyman#1069]
Keyman -- all platforms -- should have an SIL logo on splash screens and installers.

**Comment by darcywong00** [comment]
Per @mcdurdin , something along the lines of
> Keyman by SIL

Are we also including version on the splash screen?

**Comment by mcdurdin** [comment]
Yes, let's include a version on the splash screen. Here's my thinking on the proposed style. This is a mockup so images are wrong sizes, poor resolution, and aspect ratio may vary. But conceptually correct.

![image](https://user-images.githubusercontent.com/4498365/44692696-72f94800-aaa7-11e8-804c-f212ce1f8d62.png)

1. SIL logo should be slightly smaller
2. Keyman logo is good.
3. Version number and copyright text should be dynamic
4. Use copyright symbol, not `(C)` text
5. Coloured bar at bottom should be possibly twice as tall (we can stretch Keyman Team Drive:\Resources\Images\Keyman 9.0 Futago\Logo pak\keyman-banner-colours.png)

**Comment by tombogle** [comment]
Some design decisions need to be made for iOS. See the Trello card and #1161 for more details.
